src/uvw/stream.hpp

```diff
@@ -171,7 +171,9 @@ class StreamHandle: public Handle<T, U> {
     }
 
 protected:
-    using Handle<T, U>::Handle;
+	StreamHandle(std::shared_ptr<Loop> ref) noexcept
```

Comment:
Agreed: MSVC understands inherited constructors. The problem is that it gets confused easily. It complains that the constructor being inherited is ambiguous, most likely because of the multiple inheritance done by `Handle`. It seems to have no problem for other classes being compiled.
Please have a look at [the AppVeyor build report](https://ci.appveyor.com/project/tusharpm/uvw/build/job/9bklv1yv98xa6n5t/messages)([s](https://ci.appveyor.com/project/tusharpm/uvw/build/job/fi2kl0q32fraq6os/messages)) and let me know the next steps.